tests/detect_machine.sh

```diff
@@ -92,12 +92,12 @@ case $(hostname -f) in
   login4.stampede2.tacc.utexas.edu) MACHINE_ID=stampede ;; ### stampede4
 esac
 
-# Overwrite auto-detect with NEMS_MACHINE if set
-MACHINE_ID=${NEMS_MACHINE:-${MACHINE_ID}}
```

Comment:
Do we know why NEMS_MACHINE does not work?

Reply:
We renamed NEMS_COMPILER to RT_COMPILER because it's got nothing to do with NEMS (anymore). Same for machine. Thus, we try to be consistent and call it RT_MACHINE.